### PR TITLE
Update serverless page

### PR DESCRIPTION
**Pull request by pinkerton**
### What does this PR do?

Updates the serverless docs page

### Motivation

Making this simpler and to match our revamped integration docs

### Preview link

Replace the branch name and add the complete path: https://docs-staging.datadoghq.com/stephen.pinkerton/new-serverless-docs/graphing/infrastructure/serverless/

### CODE REVIEW

File: content/en/graphing/infrastructure/serverless.md [hunk 1]
```diff
@@ -27,31 +27,18 @@ The Serverless dashboard requires no installation of its own, but it relies on t
 
     This integration populates the summary graphs and the main functions table. Install this integration and ensure that Lambda metrics are reporting in your account.
 
-    **Note**: Metrics in the Cloud Function UI are delayed ~10 minutes, as this is the default speed at which Datadog polls AWS APIs. To find out if your delay can be decreased, contact [Datadog support][4].
+    **Note**: Metrics in the Serverless page are delayed ~10 minutes, as this is the default speed at which Datadog polls AWS APIs. To find out if your delay can be decreased, contact [Datadog support][4].
 
-2. [AWS X-Ray integration][5] (optional)
-    This integration provides traces for Lambda functions in the function detail page and allows the detection of Lambda function cold start.
+2. [AWS X-Ray integration][5] (recommended)
+    This integration provides full end-to-end tracing for requests hit your Lambda functions. The traces appear in the Serverless function detail page and in Datadog APM.
```
Comment (cswatt) [inline content/en/graphing/infrastructure/serverless.md]:
```suggestion
    This integration provides full end-to-end tracing for requests that hit your Lambda functions. The traces appear in the Serverless function detail page and in Datadog APM.
```

File: content/en/graphing/infrastructure/serverless.md [hunk 2]
```diff
@@ -27,31 +27,18 @@ The Serverless dashboard requires no installation of its own, but it relies on t
...
-    - Datadog's X-Ray integration supports custom subsegments and annotations as well.
+    To enable this, refer to the [AWS X-Ray integration documentation][6].
 
     **Note**: Traces are delayed ~5 minutes, as this is the speed at which Datadog polls AWS X-Ray APIs.
 
-3. AWS CloudWatch Logs (optional)
+3. AWS CloudWatch Logs (recommended)
 
-    Install this if you want to see logs from your Lambda functions in the function detail page. This also populates the additional metrics such as Memory Used (avg) and Last Start in your functions table.
+    Install this to see logs from your Lambda functions in the function detail page. This also populates the additional metrics such as Memory Used (avg) and Last Start in your functions table.
```
Comment (cswatt) [inline content/en/graphing/infrastructure/serverless.md]:
```suggestion
    Install this to see logs from your Lambda functions in the function detail page. This also populates additional metrics, such as Memory Used (avg) and Last Start in your functions table.
```

File: content/en/logs/log_collection/_index.md [hunk 3]
```diff
@@ -62,8 +62,7 @@ Choose your environment below to get dedicated log collection instructions:
 
 ## Serverless Log collection
 
-Datadog can collect logs from AWS Lambda. 
-
+Datadog can collect logs from AWS Lambda. To enable this, refer to the [AWS Lambda integration documentation][20].
```
Comment (cswatt) [inline content/en/logs/log_collection/_index.md]:
```suggestion
Datadog collects logs from AWS Lambda. To enable this, refer to the [AWS Lambda integration documentation][20].
```

File: content/en/logs/log_collection/_index.md [hunk 4]
```diff
@@ -155,7 +154,7 @@ To send logs over HTTPs for the **EU** or **US** site, refer to the [Datadog Log
 ## Datadog Logs Endpoints
 
 Datadog provides logging endpoints for both SSL-encrypted connections and unencrypted connections.
-You should use the encrypted endpoint when possible. The Datadog Agent uses the encrypted endpoint to send logs to Datadog (more information available in the [Datadog security documentation][21]).
+You should use the encrypted endpoint when possible. The Datadog Agent uses the encrypted endpoint to send logs to Datadog (more information available in the [Datadog security documentation][22]).
```
Comment (cswatt) [inline content/en/logs/log_collection/_index.md]:
```suggestion
Use the encrypted endpoint when possible. The Datadog Agent uses the encrypted endpoint to send logs to Datadog. More information is available in the [Datadog security documentation][22].
```

**Comment by pinkerton** [comment]
Let's wait to merge this until we merge the integration PR

**Comment by pinkerton** [comment]
Ready to merge

**Pull request merged**